### DISCUSSION TITLE
How does the `notch_width` property work?

### DISCUSSION THREAD

**Discussion by khoi** [FelixKratz/SketchyBar#370] [Q&A] [answered]
![khoi's screenshot 2023-05-05 at 12 35 20@2x](https://user-images.githubusercontent.com/6994441/236384098-bcdddfa3-efd8-4912-87f9-b0e0c6805e91.png)

https://github.com/khoi/dotfiles/tree/master/darwin/.config/sketchybar

I have this setup and it seems like notch handling doesn't quite work. I tried changing `notch_width=500` but there is no change.

How does this property work?

**Comment by FelixKratz** [comment, marked as answer]
The notch width only applies to the internal display and only to items that are anchored left and right of the notch (#120)

**Comment by zeidlos** [comment]
So in case anything flows under the notch, theres no way to prevent this?
Especially on smaller displays with a few (named) spaces open, its flowing under the notch and I can't even see all spaces.
